### ISSUE TITLE
Phrase selection doesn't work on MacOS

### ISSUE THREAD

**Issue by chaosarium** [gustavklopp/LingL#26]
Version: exe.macosx-10.15-x86_64-3.8 2021.06.14

MacOS maps control + click to right click, which makes selecting multiple words impossible.

**Comment by gustavklopp** [comment]
How do you multi-select on MacOS in general? (like for example on LibreOffice Calc)
Is it 'left Command'+Left click?
Have you tried the keyboard shortcut? How does the 'AltGr+T' (to focus on translation box) behaves? Is it equivalent to 'RIght command +T'?

**Comment by chaosarium** [comment]
Generally I use 'left command'+'left click' to multi select, but it seems like 'left shift'+'left click' also works. The option key on mac is usually equivalent to the Alt key.
I went back and did a bit more testing with the shortcuts. Here's what I found:
- All shortcuts without control or AltGr work as intended. 
- 'right command + T' simply opens up a new tab. I disabled this shortcut in my browser settings but it still didn't trigger 'AltGr+T'.

I tried a bunch of random key combinations with the option key and it seems like that:

- 'left option' + A focuses the "Search another spelling? [AltGr + S]" field
- 'left option' + Q marks all words in the sentence as known
- nothing happens with other combinations
